app/routes.py

```diff
@@ -1,2 +1,76 @@
-from flask import Blueprint
+from app import db 
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort 
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST"])
+def add_planet():
+    request_body = request.get_json()
+    new_planet = Planet.from_dict(request_body)
+
```

Comment:
Some some validation on request body  would be nice here to ensure required fields are present.